syncd/syncd.cpp

```diff
@@ -2169,6 +2179,10 @@ sai_status_t processEvent(
     {
         return processBulkEvent((sai_common_api_t)SAI_COMMON_API_BULK_SET, kco);
     }
+    else if (op == "bulkcreate")
+    {
+        return processBulkEvent((sai_common_api_t)SAI_COMMON_API_BULK_CREATE, kco);
```

Comment:
for new SAI "cast" will not be needed, and sairedis.h will need to be refactored to remove our implementation when new SAI headers will be in USE

Reply:
Note quite understand after several tries. Could you help after this merged?

---
In reply to: [141857082](https://github.com/Azure/sonic-sairedis/pull/233#discussion_r141857082) [](ancestors = 141857082)